assets/index.html

```diff
@@ -63,45 +65,50 @@
 <body>
     @@INCLUDE@@ navbar.html
 
-    <div class="user-info">
+    <aside class="user-info">
         <a href="/search?uploader=@_user_id_@">@_username_@</a>
         <div>
             <span class="upload">▲ @_uploaded_@</span>
             <span class="download">▼ @_downloaded_@</span>
         </div>
         <hr>
-        <div>
+        <nav>
             <a class="user-button" href="/upload">Upload</a>
             <a class="user-button" href="/invite">Invite</a>
             <a class="user-button" href="/post/create">New post</a>
-        </div>
+        </nav>
         <hr>
         <div>
             <a href="/api/user/logout">Log out</a>
         </div>
-    </div>
+    </aside>
 
-    <div class="content">
+    <aside>
```

Comment:
Seems like you lost the "content" class here, because of this the content is now at the top, while it's supposed to be at a margin from the top.